dask_jobqueue/core.py

```diff
@@ -223,9 +207,6 @@ def __init__(
         if shebang is None:
             shebang = dask.config.get("jobqueue.%s.shebang" % config_name)
 
-        if dask.config.get("jobqueue.%s.threads", None):
```

Comment:
Note: having `threads: something` in a dask config would warn and be ignored before this PR. After this PR, it will be silently ignored.